ceph/rados/rados_scrub.py

```diff
@@ -88,7 +88,7 @@ def get_pg_dump(self, *args):
         else:
```

Comment:
Which is the suite file, via which the new test added would be executed?

Reply:
The suite file is - tier-2_rados_test-scrubbing.yaml